Phonebook.Frontend/src/i18n/messages.de.xlf

```diff
@@ -401,6 +401,10 @@
         <note priority="1" from="description">Title of the Bookmarked people
             section</note>
         <note priority="1" from="meaning">DashboardComponent</note>
+      </trans-unit><trans-unit id="DashboardComponentTitleTeam" datatype="html">
+        <source> My Team </source><target state="new"> Mein Team </target>
```

Comment:
why are here spaces in front and after the text?

Reply:
format from prettier....